hca/dss/upload_to_cloud.py

```diff
@@ -27,7 +28,7 @@ def _mime_type(filename):
     return "application/octet-stream"
 
 
-def _copy_from_s3(path, s3, tx_cfg):
+def _copy_from_s3(path, s3):
```

Comment:
kind of confused why this method has this name, but probably out of the scope of this diff.